spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require "bundler/setup"
 
+require "logger"
```

Comment:
Fixes a bug with some versions of Rails: https://github.com/rails/rails/issues/54260